src/components/Careers/CareersHero/index.tsx

```diff
@@ -110,7 +110,10 @@ export const CareersHero = () => {
     const [selectedTeamName, setSelectedTeamName] = useState(teams[0])
     const selectedTeam = allTeams.find((team) => team.name.toLowerCase() === selectedTeamName.toLowerCase())
     const teamLength = selectedTeam.profiles?.data?.length
-    const teamURL = `/teams/${slugify(selectedTeam.name, { lower: true })}`
+    const teamURL = `/teams/${slugify((selectedTeam.name || '').toLowerCase().replace('ops', ''), {
+        lower: true,
+        remove: /and/,
+    })}`
```

Comment:
[This](https://github.com/PostHog/posthog.com/blob/5afbd924d75007798e380fcd56edbfe864cb24db/src/pages/teams.tsx#L71) is the matching Teams approach, where I took inspiration from.

Reply:
Nice find, thank you!